sortinghat/core/models.py

```diff
@@ -86,6 +86,9 @@ class Meta:
         db_table = 'organizations'
         unique_together = ('name',)
 
+    def __str__(self):
```

Comment:
why `__str__` instead of `__repr__` ?

Reply:
This short explanation is because `__str__` is what Django uses to display an object in the admin interface. The long answer is here https://code.djangoproject.com/ticket/710 .